docs/dialect.md

```diff
@@ -173,3 +173,110 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
 
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
+
+### Boolean Support:
+  Some databases need to customize boolean support functions.  A common case is when a database lacks native boolean support. <br/>
+  `format-true` and `format-false` are used in predicate statements. <br/>
+   **format-true** <br/> 
+  ```xml 
+   <format-true value='(1=1)' />
+  ```
+  **format-false** <br/>
+  ```xml 
+  <format-false value='(1=0)' />
+  ```
+
+**Note: Changes in Tableau 2020.3**
+
+  With the release of Tableau 2020.3, `format-true` and `format-false` will have two parameters, literal and predicate. <br/>
+  Literals is used when a boolean is used as a value. Default to: `x ? 1 : 0` <br/>
+  Example usage: `SELECT 1 as literalBool` <br/>
+  Predicates is used when a boolean is used as a predicate. Default to: `isTrue ? (1=1) : (1=0)` <br/>
```

Comment:
The wording here is a little confusing. Since there are no `x ? 1 : 0` or `isTrue ? (1=1) : (1=0)` strings anywhere in our code as far as I know, try saying "the default values are 1 and 0" and "the default values are '(1=1)' and '(1=0)'", respectively.

Reply:
Got it. This seems more clear.